sass/accordian.scss

```diff
@@ -0,0 +1,23 @@
+@import "bourbon";
+@import "neat";
+@import "mixins";
+@import "normalize";
+$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
+@import "font-awesome";
+
+
+li {
```

Comment:
Watch out about using an element selector. Use classes where possible so that you don't effect things you globally.